bundles/org.openhab.binding.webexteams/src/main/java/org/openhab/binding/webexteams/internal/WebexAuthServlet.java

```diff
@@ -0,0 +1,218 @@
+                    params.isEmpty() ? "" : String.format(HTML_META_REFRESH_CONTENT, servletBaseURL));
+            if (!StringUtil.isBlank(reqError)) {
+                logger.debug("Webex redirected with an error: {}", reqError);
+                replaceMap.put(KEY_ERROR, String.format(HTML_ERROR, reqError));
+            } else if (!StringUtil.isBlank(reqState)) {
+                try {
+                    replaceMap.put(KEY_AUTHORIZED_USER, String.format(HTML_USER_AUTHORIZED,
+                            authService.authorize(servletBaseURL, reqState, reqCode)));
+                } catch (WebexTeamsException e) {
+                    logger.debug("Exception during authorizaton: ", e);
+                    replaceMap.put(KEY_ERROR, String.format(HTML_ERROR, e.getMessage()));
+                }
+            }
+        }
+    }
+
+    /**
+     * Formats the HTML of all available Webex Accounts and returns it as a String
+     *
+     * @param accountTemplate The player template to format the account values in
+     * @param servletBaseURL the redirect_uri to be used in the authorization url created on the authorization button.
+     * @return A String with the accounts formatted with the account template
+     */
+    private String formatAccounts(String accountTemplate, String servletBaseURL) {
+        final List<WebexTeamsHandler> accounts = authService.getWebexTeamsHandlers();
+
+        return accounts.isEmpty() ? HTML_EMPTY_ACCOUNTS
+                : accounts.stream().map(p -> formatAccount(accountTemplate, p, servletBaseURL))
+                        .collect(Collectors.joining());
+    }
+
+    /**
+     * Formats the HTML of a Webex Account and returns it as a String
+     *
+     * @param accountTemplate The account template to format the player values in
+     * @param handler The handler for the account to format
+     * @param servletBaseURL the redirect_uri to be used in the authorization url created on the authorization button.
+     * @return A String with the account formatted with the account template
+     */
+    private String formatAccount(String accountTemplate, WebexTeamsHandler handler, String servletBaseURL) {
+        final Map<String, String> map = new HashMap<>();
+
+        map.put(ACCOUNT_ID, handler.getUID().getAsString());
+        map.put(ACCOUNT_NAME, handler.getLabel());
+        final String webexUser = handler.getUser();
+
+        if (!handler.isConfigured()) {
+            map.put(ACCOUNT_USER_ID, "");
+            map.put(ACCOUNT_SHOWBTN, "u-hide");
+            map.put(ACCOUNT_SHWOMSG, "u-show");
+            map.put(ACCOUNT_MSG, "Configure account.");
+        } else if (handler.isAuthorized()) {
+            map.put(ACCOUNT_USER_ID, String.format(" (Authorized user: %s)", webexUser));
+            map.put(ACCOUNT_SHOWBTN, "u-hide");
+            map.put(ACCOUNT_SHWOMSG, "u-show");
+            map.put(ACCOUNT_MSG, "Authorized.");
+        } else if (!StringUtil.isBlank(webexUser)) {
```

Comment:
You can simply use `!webexUser.isBlank()` ?

Reply:
Updated. And removed import of StringUtil.